pkg/sql/schema_changer.go

```diff
@@ -416,15 +434,15 @@ func (sc *SchemaChanger) exec(ctx context.Context) error {
 	}
 	if notFirst {
 		log.Infof(ctx,
-			"schema change on %s (%d v%d) mutation %d: another change is still in progress",
-			tableDesc.Name, sc.tableID, tableDesc.Version, sc.mutationID,
+			"schema change on %s (v%d): another change is still in progress",
```

Comment:
Should the `%s` here be `%q`, as it is elsewhere? (Also below)

Reply:
Good point, I changed it.